kustomize/move2kube/README.md

```diff
@@ -0,0 +1,78 @@
+## Prerequisites
+* The manifests are generated without the `namespace` configuration
+* Manifest names reflect the `resources` section of [kustomization.yaml](./base/kustomization.yaml)
+
+A brief explanation of the system architecture is provided in the related [document](./move2kube.md).
+
+## Kustomization options
+### Image update
+Run the following to update the default images to a custom configuration
+```
+cd base && kustomize edit set image serverless-workflow-move2kube=quay.io/orchestrator/serverless-workflow-move2kube:1234 && cd ..
+cd base && kustomize edit set image serverless-workflow-m2k-kfunc=quay.io/orchestrator/serverless-workflow-m2k-kfunc:1234 && cd ..
+```
+
+## Configure properties
+Edit configuration in [replace-config.yaml](./base/replace-config.yaml), [config.properties](./base/config.properties) and 
+[secret.properties](./base/secret.properties) to match your environment configuration.
+
+## Deploy to the cluster
+### Deploy the production environment
+This environment applies the generated manifests with minimal customizations to:
+* Use the latest application images
+* Force the `prod` profile for the SonataFlow instance
+* Deploy by default on the `sonataflow-infra` namespace
+* Mount the configurations defined in [Configure properties](#configure-properties) as environment variables
+
+The default namespace can be customized with:
+```bash
+TARGET_NS=YOUR-NS
+cd base && kustomize edit set namespace=$TARGET_NS && cd ..
```

Comment:
I would split that in 2 so I can just click on the execute button of the second command :)

Reply:
Is this command right, I have the following:
```
$ kustomize edit set namespace=${TARGET_NS}
Sets the value of different fields in kustomization file.

Usage:
  kustomize edit set [command]

Examples:

        # Sets the nameprefix field
        kustomize edit set nameprefix <prefix-value>

        # Sets the namesuffix field
        kustomize edit set namesuffix <suffix-value>


Available Commands:
  annotation  Sets one or more commonAnnotations in kustomization.yaml
  image       Sets images and their new names, new tags or digests in the kustomization file
  label       Sets one or more commonLabels in kustomization.yaml
  nameprefix  Sets the value of the namePrefix field in the kustomization file.
  namespace   Sets the value of the namespace field in the kustomization file
  namesuffix  Sets the value of the nameSuffix field in the kustomization file.
  replicas    Sets replicas count for resources in the kustomization file

Flags:
  -h, --help   help for set

Global Flags:
      --stack-trace   print a stack-trace on error

Use "kustomize edit set [command] --help" for more information about a command.
```

Reply:
So it should be 
``
$ kustomize edit set namespace ${TARGET_NS} 
```